### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by sarbull**
Signed-off-by: Sirbu Nicolae-Cezar <sirbunicolaecezar@gmail.com>

### CODE REVIEW

**Comment by sarbull** [comment]
@planetf1 let's continue the discussion here

**Comment by sarbull** [comment]
@planetf1 i updated the PR here

**Review by planetf1 — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
+FROM node:16.13.0 AS builder
 
-ARG version=1.4.0
+WORKDIR /app
+COPY . /app
+RUN npm install
+RUN npm run build:docker
+
+# Stage 2 - Containerize
+FROM nginx:1.19.0
```
[the remark this answers is not in the document]
Comment (planetf1) [reply Dockerfile:13]: Alternatively could try https://github.com/odpi/egeria-ui/issues/289 - that would be the desired end-state, in particular for runtime

**Review by planetf1 — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
+
+ARG version=4.0.2
```
[the remark this answers is not in the document]
Comment (planetf1) [reply Dockerfile:15]: No I think you are right to use a fixed version!

File: Dockerfile [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+FROM node:16.13.0 AS builder
 
-ARG version=1.4.0
+WORKDIR /app
+COPY . /app
+RUN npm install
+RUN npm run build:docker
+
+# Stage 2 - Containerize
+FROM registry.access.redhat.com/ubi8/nginx-120
```
[the remark this answers is not in the document]
Comment (planetf1) [reply Dockerfile:13]: 1.71 is the latest stable @sarbull ( https://catalog.redhat.com/software/containers/ubi8/nginx-120/6156abfac739c0a4123a86fd )

[the remark this answers is not in the document]
Comment (sarbull) [reply Dockerfile]: updated

**Review by planetf1 — APPROVED** [review]

**Pull request merged**